langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -301,6 +434,19 @@ public isolated function shift(Type[] arr) returns Type = @java:Method {
 # The values newly added to the array will be in the same order
 # as they are in parameter `vals`.
 #
+# ```
+# int[] evenNumbers = [14];
+#
+# // Add multiple members to the start of the list.
+# evenNumbers.unshift(10, 12);
+# evenNumbers => [10,12,14]
```

Comment:
need spaces

Reply:
This is the toString representation, which is also what the shell produces.